src/components/Customers.js

```diff
@@ -88,24 +90,72 @@ class Customers extends React.Component {
 			data: this.sort(this.state.data, columnName),
 			sortDirection: foo
 		});
+	}
 
-		// console.log('after', this.state.data)
+	handleClick = (event) => {
+		console.log(event.currentTarget)
+		// console.log(event)
+		if(this.state.isRowClicked === false) {
+			this.setState({
+				isRowClicked: true,
+				counter: ++this.state.counter
+			});
+		} else {
+			this.setState({
+				isRowClicked: false,
+				counter: --this.state.counter
+			});
+		}
 	}
 
 	render() {
-		// console.log(this.state.columnToSort, this.state.sortDirection);
-		return (
-			<div>
-				<p>Find Customers: 
-					<button className='btn btn-primary' onClick={this.getData}>Find</button>
-				</p>
-				<table className="table">
-					<TableHead sort={this.handleSort} />
-					<TableData customers={this.state.data} 
-					/>
-				</table>
-			</div>
-		);
+		// console.log('error message here', this.state.error)
+		if(this.state.error) {
+			return (
+				<p>There is an error: <span>{this.state.error}</span></p>
+			);
+		} else {
+			return (
+				<div>
+					<section className="get-results">
+
+						{/*SPINNER SECTION*/}
+	          <section className={this.state.isLoading ? 'spinner' : 'hidden'}>
```

Comment:
Maybe this would be worth extracting out?

And I would usually just remove the spinner from the DOM (i.e. not render it) if we're not loading. It's simpler to do this than hide with CSS